modules/cli/src/test/java/org/apache/ignite/internal/cli/commands/flow/FlowTest.java

```diff
@@ -269,24 +288,27 @@ void customDecorator() throws IOException {
         assertThat(errOut.toString(), emptyString());
     }
 
-    private static Flow<Object, Integer> createFlow() {
-        return askQuestion()
-                .question(s -> "Here is your number " + s + ":, would you like to multiply it by 2?",
-                        List.of(new QuestionAnswer<>("yes"::equals, (a, i) -> Integer.parseInt(i) * 2),
-                                new QuestionAnswer<>("no"::equals, (a, i) -> Integer.parseInt(i))))
-                .ifThen(num -> num == 1, Flows.fromCall(new IntCall(), IntCallInput::new))
-                .ifThen(num -> num > 1, Flows.fromCall(new StrCall(), integer -> new StrCallInput(String.valueOf(integer))))
-                .build();
+    @Test
+    void flatMap() {
+        Flows.from("fizz")
+                .flatMap(v -> Flows.from(it -> it + "buzz"))
+                .print()
+                .start();
+        assertThat(out.toString(), equalTo("fizzbuzz" + System.lineSeparator()));
     }
 
-    private static FlowBuilder<Object, String> askQuestion() {
-        return Flows.question("Do you like this?",
-                        List.of(new QuestionAnswer<>("yes"::equals, (a, i) -> 1),
-                                new QuestionAnswer<>("no"::equals, (a, i) -> 2))
-                )
-                .map(String::valueOf);
+    @Test
+    void flatMapInterrupted() {
+        Flows.from("fizz")
```

Comment:
I would like to see more complex tests, like:
 ```
Flows.from("fizz")
   .map(v -> Flows.from(it -> it + "1"))
   .print()
   .flatMap(v -> {
                     throw new FlowInterruptException();
                 })
  .print()
  .map(v -> Flows.from(it -> it + "2"))
  .print()
  .start();
```

Reply:
done